pipeline-modules/app-service/gcp/go-build.yaml

```diff
@@ -25,10 +25,19 @@ inputs:
       description: The current working directory to execute command
       type: string
       default: repo
+    go_version:
+      title: Golang version
+      description: The version of Golang to use
+      type: string
+      default: latest
 template: |
   steps:
-  - id: 'Go build'
-    name: 'gcr.io/cloud-builders/go'
+  - id: 'Go Version'
+    name: "gcr.io/cloud-builders/go:{{ go_version }}"
+    args: ["version"]
+    env: ["GOPATH=/workspace"]
+  - id: 'Go Build'
+    name: 'gcr.io/cloud-builders/go:{{ go_version }}'
     dir: {{ working_dir }}
     env: ["GOPATH=/workspace", "PROJECT_ROOT={{ working_dir }}"]
```

Comment:
Shouldn't we also explicitly default CGO_ENABLED to 0 as in the above module?

Reply:
sure - it is currently defaulted by google's build agent to 0

Reply:
This was fixed